src/helper_input.R

```diff
@@ -61,6 +61,7 @@ collapseStory <- function() {
                              "Current or former smoker" = "is_smoker",
                              "Immunocompromised condition" = "is_immune",
                              "Chronic lung disease or asthma" = "is_lung",
+                             "Obesity" = "is_obesity",
```

Comment:
Let's change the label to "Obesity (BMI >=30kg/m2)" I don't know Rshiny's ability in rendering math equations and superscripts, but should be doable.